### ISSUE TITLE
Direct shortcut for turn on/off VPN Mode or System Proxy

### ISSUE THREAD

**Issue by Saeed-Tavoosi** [MatsuriDayo/nekoray#213]
**描述建议**

**建议的必要性**

You added a shortcut to open a pop up, for select VPN Mode or System Proxy,
What do you think about direct shortcut to turn on/off VPN Mode or System Proxy?

**Comment by v-JiangNan** [comment]
Yes, my feature request in #182  are exactly what you said , but the developer may have misunderstood, or I may not have expressed them clearly.
